pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -751,6 +752,106 @@ func (k *Bootstrapper) restartPrimaryControlPlane(cfg config.ClusterConfig) erro
 	return nil
 }
 
+func (k *Bootstrapper) SetMinikubeFolderErrorScript(hostDriverIP string) (string, error) {
+	out.Step(style.Provisioning, "Setting up minikube certificates folder...")
+	script := fmt.Sprintf(
+		`mkdir c:\var\lib\minikube\certs; Copy-Item C:\etc\kubernetes\pki\ca.crt -Destination C:\var\lib\Minikube\Certs; Remove-Item C:\etc\kubernetes\pki\ca.crt`)
```

Comment:
cert directories may be better suited as variables since the script references the same folders twice, easier to maintain as well.

Reply:
Have the cert directories as variables!